scripts/platform-migrate.sh

```
@@ -0,0 +1,470 @@
+                    echo "railway variables set $key=\"$value\""
+                fi
+            done
+            ;;
+        render)
+            print_info "Set environment variables in Render dashboard manually"
+            print_info "Backup file: $env_file"
+            ;;
+        heroku)
+            local app_name="masterspeak-$ENVIRONMENT"
+            print_info "Setting Heroku environment variables..."
+            source "$env_file"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)